src/test/end-to-end/tests/C100Rebuild-BaseFlow.js

```diff
@@ -0,0 +1,21 @@
+Feature('C100 Rebuild - Base flow');
```

Comment:
Are we executing the test case for only one flow? We have 4 types of flow right?

Reply:
Due to time constraints, I have only done the base flow. I will have to do the rest 3 flows after New Years